06-developer-software-guide.Rmd

```diff
@@ -99,163 +78,109 @@ Here are some that you may want to use with FIMS:
 }
 ```
 
-Some [suggested R
-shortcuts](https://github.com/REditorSupport/vscode-R/wiki/Keyboard-shortcuts)
-could be helpful.
-
-
-To set up C++ with vscode, instructions are 
-[here](https://code.visualstudio.com/docs/languages/cpp).
-
-Other helpful extensions that can be found in the VScode marketplace are:
- - Github Copilot: An AI tool that helps with line completion
- - Live Share: Collaborate on the same file remotely with other developers
- - Rewrap: Helps rewrapping comments and text lines at a specified character
-   count. Note that to get it working it will be necessary to add
-   [rulers](https://code.visualstudio.com/docs/getstarted/tips-and-tricks#_vertical-rulers)
- - There are a number of keymap packages that import key mappings from commonly
-  used text editors (e.g., Sublime, Notepad++, atom, etc.). Searching "keymap"
-  in the marketplace should help find these.
-  - GitLens (or GitLess): Adds more Git functionality. Note that some of the
-    GitLens functionality is not free, and GitLess is a fork before the addition
-    of these premium features.
-
-Note that the keybindings.json and settings.json could be copied from one
-computer to another to make it easier to set up VS code with the settings
-needed. Note that the [settings.json location differs](https://code.visualstudio.com/docs/getstarted/settings#_settings-file-locations) depending on the operating system.
-
-Typically, it is good practice to not restore old sessions after shutting down
-the IDE. To avoid restoring old sessions in the VS Code terminals (including R
-terminal), in the Setting User Interface within VS Code (get to this by opening
-the command palette and searching for Preferences: Open Settings (UI)), under
-Features > Terminal, uncheck the option "Enable Persistent Sessions."
-
-Rstudio addins can be accessed by searching for Rstudio addin in the command
-palette. Clicking on "R: Launch Rstudio Addin" should provide a list of addin
-options.
+Other helpful VS Code extensions are available in the VScode marketplace, e.g.,
+  - Github Copilot: An AI tool that helps with line completion.
+  - Live Share: Collaborate on the same file remotely with other developers.
+  - Rewrap: Helps rewrapping comments and text lines at a specified character count. Note that to get it working it will be necessary to add [rulers](https://code.visualstudio.com/docs/getstarted/tips-and-tricks#_vertical-rulers).
+  - GitLens (or GitLess): Adds more Git functionality. Some of the GitLens functionality is not free, and GitLess is a fork before the addition of these premium features.
+  - Selecting "R: Launch Rstudio Addin" from the command pallete will provide a list of addin options for RStudio.
```

Comment:
I was confused about why this line was in the VS Code section, but it seems that they are RStudio addins for VS Code (to make VS Code work more like RStudio). That could be clarified.

Reply:
Reworded to "A list of extensions that can help make VS Code operate more like RStudio is available by typing `R: Launch Rstudio Addin' in the command pallete."